std/portable.d.ts

```diff
@@ -129,6 +129,11 @@ declare function store<T = u8>(ptr: usize, value: T, constantOffset?: usize): vo
 /** Emits an unreachable operation that results in a runtime error when executed. */
 declare function unreachable(): any; // sic
 
+/** [Polyfill] Performs the sign-agnostic reverse bytes **/
+declare function bswap<T = i8 | u8 | i16 | u16 | i32 | u32 | isize | usize>(value: T): T;
+/** [Polyfill] Performs the sign-agnostic reverse bytes only for last 16-bit **/
+declare function bswap16<T = i8 | u8 | i16 | u16 | i32 | u32>(value: T): T;
```

Comment:
Do these actually exist in the portable JS library (portable.js)? If not, you can just skip them in portable.d.ts.

Reply:
Ok, I implement this in portable.js

Reply:
Note that in portable, you have to decide for let's say i32 arguments only.

Reply:
Done